tests/test_focal_loss.py

```diff
@@ -104,17 +104,22 @@ def test_multi_class_seg_2d(self):
         # add another dimension corresponding to the batch (batch size = 1 here)
         target = target.unsqueeze(0)  # shape (1, H, W)
         pred_very_good = 1000 * F.one_hot(target, num_classes=num_classes).permute(0, 3, 1, 2).float()
-
         # initialize the mean dice loss
         loss = FocalLoss()
 
         # focal loss for pred_very_good should be close to 0
+        target_one_hot = F.one_hot(target, num_classes=num_classes).permute(0, 3, 1, 2)  # test one hot
         target = target.unsqueeze(1)  # shape (1, 1, H, W)
+
         focal_loss_good = float(loss.forward(pred_very_good, target).cpu())
```

Comment:
Suggest using the "\__call__()" API instead of `forward()`, it's the formal API in PyTorch nn.Module:
```py
focal_loss_good = float(loss(pred_very_good, target).cpu())
```

Reply:
agreed. not sure why @myron changed it to `.forward()` in a previous discussion https://github.com/Project-MONAI/MONAI/pull/487#discussion_r436186238

Reply:
hmm, I also think we should just use __call__ api.  I used .forward in test_masked_dice_loss.py, just because test_dice_loss.py was using it.